scripts/pre_commit_linter.py

```diff
@@ -984,6 +986,12 @@ def _check_docstrings(all_files):
                 if line_num > 0:
                     prev_line = file_content[line_num - 1].lstrip().rstrip()
 
+                # Check for single line docstring starting with a space
```

Comment:
Remove `single line` from the comment since the check is same for both of single line as well as multi line docstring.

Reply:
Done!